sdk/communication/communication-administration/package.json

```diff
@@ -9,8 +9,7 @@
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
-    "build:autorest": "npm run build:autorest-identity && npm run build:autorest-phone",
-    "build:autorest-identity": "autorest --typescript --version=3.0.6267  --v3 ./swagger/CommunicationIdentity.md && rushx format",
+    "build:autorest": "npm run build:autorest-phone",
```

Comment:
nit: build:autorest-phone no longer needed

Reply:
I'm aware of the nit :-)
We'll delete all of administration, so I didn't bother. Also, autorest doesn't have to generate into a phoneNumbers subfolder anymore.